cloud/deploy/deploy.go

```diff
@@ -102,9 +103,8 @@ func getRegistryURL(domain string) string {
 	return registry
 }
 
-func deployDags(path, runtimeID string, client astro.Client) error {
+func deployDags(path, dagsPath, runtimeID string, client astro.Client) error {
```

Comment:
WDYT of naming this as `dagsFolderPath`?

Reply:
keeping it to dagsPath as it being referred to so in the existing code for `astro deploy` and would not like to rename it in this PR.